addons/mail/security/mail_security.xml

```diff
@@ -69,6 +107,20 @@
             <field name="domain_force">[(1, '=', 1)]</field>
         </record>
 
+        <record id="discuss_gif_favorite_user_rule" model="ir.rule">
+            <field name="name">Discuss.gif.favorite: User access</field>
+            <field name="model_id" ref="model_discuss_gif_favorite"/>
+            <field name="groups" eval="[Command.link(ref('base.group_user'))]"/>
+            <field name="domain_force">[('create_uid', '=', user.id)]</field>
+        </record>
```

Comment:
This rule should allow you to remove the `create_uid` condition on the get_favorite method from the gif controller?

Reply:
It's just a move so I don't know, it means it would be like that already. I completely ignored other models than channel and member in this PR.